app/services/idv/phone_step.rb

```diff
@@ -106,7 +106,7 @@ def update_idv_session
       idv_session.address_verification_mechanism = :phone
       idv_session.applicant = applicant
       idv_session.vendor_phone_confirmation = true
-      idv_session.user_phone_confirmation = phone_matches_user_phone?
+      idv_session.user_phone_confirmation = false
```

Comment:
I wonder if there are other things we could simplify if we can always assume OTP confirmation will be required. At first I wondered if we could get rid of this variable altogether, though it appears it's necessary to push people to the review step if they're already done with this one.

Maybe we could still clean up some of the OTP-related redirect logic?

e.g.

https://github.com/18F/identity-idp/blob/a6a6a087ae96b92dfb4608c3f116fb8472427784/app/controllers/idv/phone_controller.rb#L54

Reply:
Potentially?  I'm not sure though, the current flow is:

1. PhoneFinder
2. Confirm Phone via OTP if needed

so the stuff we can delete in the app might be more limited.  The tests are a different story though.